connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java

```diff
@@ -74,6 +75,7 @@ public WorkerCoordinator(ConsumerNetworkClient client,
                              WorkerRebalanceListener listener) {
         super(client,
                 groupId,
+                sessionTimeoutMs, // FIXME: What should Connect use for the rebalance timeout?
```

Comment:
Connect has a lot of the same problems as a consumer app has wrt potentially long operations. During rebalance, we need to stop all connectors and commit offsets, which can also include flushing outstanding data to external systems. This can potentially take a long time (and I think is still serial, unfortunately). It was probably an oversight that we didn't include the option for Connect in the KIP. We include the non-consumer related configs already (session timeout and heartbeat interval), so it makes sense to me to expose this as well. An operator would increase it if they know they will have a connector plugin that can take a long time to flush data.